src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('span.population')].map(
+  (el) => +el.textContent.replace(/,/g, ''),
+);
+
+const total = population.reduce((a, b) => a + b);
+const average = Math.floor(total / population.length);
```

Comment:
There is a potential issue here if the 'population' array is empty. This would result in a division by zero error when calculating the average. Consider adding a check to ensure that the 'population' array is not empty before performing the division.